### PR TITLE
DL-2663 - Updated dependencies

### PR DESCRIPTION
**Pull request by alijy**
# DL-2663 - Updated dependencies

**New feature**

Updated dependencies

## Checklist

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Review by benryan1790 — CHANGES_REQUESTED** [review]
Following libraries/plugins can be upgraded further:

- com.kenshoo:metrics-play
- org.mockito:mockito-core
- org.scalatest:scalatest
- org.scalatestplus.play:scalatestplus-play
- org.scoverage:sbt-scoverage

**Comment by alijy** [comment]
> Following libraries/plugins can be upgraded further:
> 
> * com.kenshoo:metrics-play
> * org.mockito:mockito-core
> * org.scalatest:scalatest
> * org.scalatestplus.play:scalatestplus-play
> * org.scoverage:sbt-scoverage

Done. (Only scoverage needed an upgrade)

File: buid.sbt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import uk.gov.hmrc.DefaultBuildSettings.targetJvm
```
Comment (stephen-oppong-beduh) [inline buid.sbt]: The build.sbt file is named ```buid.sbt```

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+object AppDependencies {
+
+  private val typesafe = "com.typesafe.play"
+  private val hmrc = "uk.gov.hmrc"
+
+  val compile: Seq[ModuleID] = Seq(
+    ws,
+    hmrc %% "bootstrap-play-26" % "1.3.0",
+    "com.github.fge" % "json-schema-validator" % "2.2.6",
+    typesafe %% "play-json-joda" % "2.7.4"
```
Comment (stephen-oppong-beduh) [inline project/AppDependencies.scala]: ```play-json-joda``` can go up to 2.8.1

Comment (alijy) [reply project/AppDependencies.scala]: Done.

**Review by stephen-oppong-beduh — APPROVED** [review]

**Pull request merged**